tests/network/bindingplugin_passt.go

```diff
@@ -63,12 +64,20 @@ var _ = SIGDescribe("[Serial] VirtualMachineInstance with passt network binding
 
 	BeforeEach(func() {
 		const passtBindingName = "passt"
+
+		var passtComputeMemoryOverheadWhenAllPortsAreForwarded = resource.MustParse("500Mi")
```

Comment:
Please update the commit message with this value.

Reply:
It is already there in the last sentence.

Reply:
`800Mi` is mentioned twice in the commit message.

Reply:
The last sentence says:
> According to passt's maintainer, we could now use 500Mi.

Reply:
My bad, fixed the first sentence.